yarn.lock

```diff
@@ -8346,58 +8346,6 @@ doctrine@^3.0.0:
   dependencies:
     esutils "^2.0.2"
 
-docusaurus@^2.0.0-alpha.61:
```

Comment:
should not be needed anymore after updating from master

Reply:
done